sdk/eventgrid/azure-messaging-eventgrid/src/test/java/com/azure/messaging/eventgrid/DeserializationTests.java

```diff
@@ -208,7 +181,7 @@ public void consumeStorageBlobDeletedEventWithExtraProperty() throws IOException
         EventGridEvent[] events = EventGridEvent.fromString(jsonData).toArray(new EventGridEvent[0]);
 
         assertNotNull(events);
-        assertTrue(toSystemEventData(events[0]) instanceof StorageBlobDeletedEventData);
+        assertInstanceOf(StorageBlobDeletedEventData.class, toSystemEventData(events[0]));
```

Comment:
`Assertions.assertInstanceOf` returns a value of the object mapped to the type if you want to further cleanup code by removing the type cast in the line below.